source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -52,10 +50,22 @@ Host: oidc.integration.account.gov.uk
 
 <%= warning_text('This code example uses formatting that makes it easier to read. If you copy the example, you must make sure the request is:<ul><li>a continuous line of text separating each parameter with an ampersand (&)</li><li>not split across multiple lines</li><li>without any additional separators such as newline, commas or tabs</li></ul>') %>
 
-### Make an authorisation request for authentication and identity
-If you need identity assurance as well as authentication, use this example message to make a `GET` request.
+### Make a request for authentication and identity
+
+If you need to authenticate and process identity data, you should send 2 separate requests, one for authentication and one for identity. 
```

Comment:
Happy to change this, but just for info, I used 'process' because of the title of our section in the tech docs: https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/process-identity-information/

Reply:
Yes, I think we're OK to use the suggestion in that case - the processing bit is a route to the main goal. But that's a good point and I think it's worth thinking (later) about whether we should have headings for 'authenticate your user' and 'check your user's identity' in the integrate section. Then the other headings would sit under them to make the end goal clearer.